sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-shared.service.ts

```diff
@@ -350,6 +350,7 @@ export class DisbursementScheduleSharedService extends RecordDataModelService<Di
           },
         },
         disbursementScheduleStatus: In(status),
+        hasEstimatedAwards: true,
```

Comment:
This method is used by the overawards section to `cancel pending disbursements`  and `sum DisbursedValues Per Value Code`

![image](https://github.com/user-attachments/assets/e2a9b1d9-cd7d-46bc-b347-3b6f2189aae7)

![image](https://github.com/user-attachments/assets/d9595a43-afde-457a-ab2c-1200634904da)

I believe the condition is not required here because of 2 reasons.

1. When an assessment is cancelled or reassessment happens we must continue to cancel any pending disbursement(s) whether it has disbursement amount or not.
2. Calculation of sum of disbursed values will not be affected by this condition being there.

@andrewsignori-aot thoughts??

Reply:
Yes, the `cancelPendingDisbursements` still need to be able to get the records to be canceled.
There is a real issue with `sumDisbursedValuesPerValueCode` because it deals only with "ready/sent" but still the check `hasEstimatedAwards: true` should be removed as recommended by @dheepak-aot.

Reply:
Thanks @dheepak-aot @andrewsignori-aot for giving the clarification. It totally makes sense. I'll remove `hasEstimatedAwards: true` from the query.